packages/manager/modules/pci/src/projects/project/storages/databases/database/users/add/add.html

```diff
@@ -3,83 +3,87 @@
     data-on-click="$ctrl.goBack()"
 ></oui-back-button>
 
-<h2 data-translate="pci_databases_users_add"></h2>
-<oui-field
-    data-label="{{:: 'pci_databases_users_add_username_label' | translate }}"
-    data-size="xl"
->
-    <input
-        class="oui-input"
-        type="text"
-        id="username"
-        name="username"
-        ng-model="$ctrl.model.username"
-        maxlength="255"
-        required
-    />
-</oui-field>
-
-<div ng-if="!$ctrl.isRolesReadOnly">
-    <h2 data-translate="pci_databases_users_add_roles"></h2>
-    <oui-dual-list
-        data-source="$ctrl.availableRoles"
-        data-target="$ctrl.model.selectedRoles"
-        disabled="$ctrl.isFeatureActivated('getRoles')"
-        property="name"
+<form novalidate>
+    <h2 data-translate="pci_databases_users_add"></h2>
+    <oui-field
+        data-label="{{:: 'pci_databases_users_add_username_label' | translate }}"
+        data-size="xl"
     >
-        <oui-dual-list-source
-            data-heading="{{:: 'pci_databases_users_add_roles_available' | translate }}"
-            data-searchable="true"
-        >
-        </oui-dual-list-source>
-        <oui-dual-list-target
-            data-heading="{{:: 'pci_databases_users_add_roles_selected' | translate }}"
-            data-searchable="true"
+        <input
+            class="oui-input"
+            type="text"
+            id="username"
+            name="username"
+            ng-model="$ctrl.model.username"
+            data-ng-pattern="$ctrl.NAME_PATTERN"
+            data-ng-maxlength="$ctrl.NAME_MAX_LENGTH"
+            required
+        />
+    </oui-field>
+
+    <div ng-if="!$ctrl.isRolesReadOnly">
```

Comment:
isReadOnly or isReadOnlyUser?? just to add more meaning.

Reply:
It is not the whole user that is in readonly mode, only his roles. Other inputs on the page are still editable, so I cannot say that the whole page or user is readonly :wink: